core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java

```diff
@@ -111,215 +164,221 @@
+    
     // login contact
-    public static final String SKIN_LOGIN_CONTACT_HTML = "skin.login.contact_html";
-    public static final String DEFAULT_SKIN_LOGIN_CONTACT_HTML = "If you think you have received this message in " +
+    @Value("${skin.login.contact_html}")
+    private static String skinLoginContactHtml;
+    private static final String DEFAULT_SKIN_LOGIN_CONTACT_HTML = "If you think you have received this message in " +
             "error, please contact us at <a style=\"color:#FF0000\" href=\"mailto:cbioportal-access@cbio.mskcc.org\">" +
             "cbioportal-access@cbio.mskcc.org</a>";
-
+    
     // properties for hiding/showing tabs in the patient view
-    public static final String SKIN_PATIENT_VIEW_SHOW_CLINICAL_TRIALS_TAB="skin.patient_view.show_clinical_trials_tab";
-    public static final String SKIN_PATIENT_VIEW_SHOW_DRUGS_TAB="skin.patient_view.show_drugs_tab";
-
+    @Value("${skin.patient_view.show_clinical_trials_tab:false}")
+    private static boolean skinPatientViewShowClinicalTrialsTab;
+    
+    @Value("${skin.patient_view.show_drugs_tab:false}")
+    private static boolean skinPatientViewShowDrugsTab;
+    
     // property for setting the saml registration html
-    public static final String SKIN_LOGIN_SAML_REGISTRATION_HTML = "skin.login.saml.registration_html";
-    public static final String DEFAULT_SKIN_LOGIN_SAML_REGISTRATION_HTML = "Sign in with MSK";
-
+    @Value("${skin.login.saml.registration_html}")
+    private static String skinLoginSamlRegistrationHtml;
+    private static final String DEFAULT_SKIN_LOGIN_SAML_REGISTRATION_HTML = "Sign in with MSK";
+    
     // property for the saml entityid
-    public static final String SAML_IDP_METADATA_ENTITYID="saml.idp.metadata.entityid";
+    @Value("${saml.idp.metadata.entityid}")
+    private static String samlIdpMetadataEntityId;    
 
     // property for the custom header tabs
-    public static final String SKIN_CUSTOM_HEADER_TABS="skin.custom_header_tabs";
-
+    @Value("${skin.custom_header_tabs}")
+    private static String[] skinCustomHeaderTabs;
+        
     // properties for the FAQ, about us, news and examples
-    public static final String SKIN_BASEURL="skin.documentation.baseurl";
-    public static final String DEFAULT_SKIN_BASEURL="https://raw.githubusercontent.com/cBioPortal/cbioportal/master/docs/";
-    public static final String SKIN_DOCUMENTATION_MARKDOWN="skin.documentation.markdown";
+    @Value("${skin.documentation.baseurl}")
+    private static String skinDocumentationBaseUrl;
+    private static final String DEFAULT_SKIN_BASEURL = "https://raw.githubusercontent.com/cBioPortal/cbioportal/master/docs/";
+    
+    @Value("${skin.documentation.markdown:true}")
+    private static boolean skinDocumentationMarkdown;
+    
+    @Value("${skin.documentation.faq}")
+    private static String skinDocumentationFaq;
+    private static final String DEFAULT_SKIN_FAQ = "FAQ.md";
+    
+    @Value("${skin.documentation.about}")
```

Comment:
@pieterlukasse I used the default value for some properties where the strings/values are much shorter text and so I could do the same for the case above that you commented on as well of course. 

I was wondering though, for the prop's with much longer default text, should I leave the default value setting to the methods that call these properties?